annotation/BUILD

```diff
@@ -0,0 +1,12 @@
+# Description:
```

Comment:
The BUILD files don't get synced at all right now. It'd be nice to converge them, but that's a ways off.